MessageManager.class.nut

```diff
@@ -120,12 +121,15 @@ class MessageManager {
     // Callback to be executed when a partner responds to the "connected" notification
     _onConReply = null;
 
+    // First message ID
```

Comment:
I still don't think that we need this variable at the class level. It's a local variable in the scope of the constructor.